docs/excel/custom-functions-overview.md

```diff
@@ -360,6 +360,50 @@ function secondHighest(values){
 }
 ```
 
+## Discovering cells that invoke custom functions
+
+You may wish to discover the cell that invoked a custom function. Common scenarios for using `caller.address` include:
+
+- Formatting ranges: You can use `caller.address` as the key of the cell to store information in [`AsyncStorage`](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-runtime#storing-and-accessing-data). Then, use [the `onCalculated` event](https://docs.microsoft.com/en-us/javascript/api/excel/excel.worksheet?view=office-js#oncalculated) in Excel to load the key from `AsyncStorage`.
+- Displaying cached values: If your function is being used offline, you can display stored cached values from `AsyncStorage`. Display using the `onCalculated` event.
+- Reconciliation: You can use `caller.address` to discover an origin cell to help you reconcile where processing is occurring.
```

Comment:
Is reconciliation something people know about going in? I'm unclear what process is being described here.

Reply:
I will check with Keyur - my impression is that yes, it is something that the CF people are knowledgeable about but agree with you that it is confusingly worded.

Reply:
@keyur32 thoughts on this one?